flank-bash/scripts/updateLibs/updateSwift.main.kts

```diff
@@ -0,0 +1,77 @@
+@file:CompilerOptions("jvmTarget = '1.8'")
+
+@file:Import("../util/downloadFiles.main.kts")
+@file:Import("../util/archive.main.kts")
+@file:Import("../util/OsHelper.kts")
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+private val currentPath = Paths.get("")
+private val swiftPath = Paths.get(currentPath.toString(), "swift")
+
+fun updateSwiftWindows() {
+    val swiftExe = Paths.get(swiftPath.toString(), "swift.exe")
+
+    if (swiftExe.toFile().exists()) {
+        println("Swift exists")
+    } else {
+        println("Downloading swift for Windows")
+        swiftPath.toFile().mkdirs()
+        downloadFile(
+            url = "https://swift.org/builds/swift-5.3-release/windows10/swift-5.3-RELEASE/swift-5.3-RELEASE-windows10.exe",
+            destinationPath = swiftExe.toString()
+        )
+    }
+
+    swiftExe.toFile().extract(swiftPath.toFile(), "zip", "gz")
```

Comment:
same as above

Reply:
@Sloox I think that you just test this for #1134 ticket, if yes we should remove this code and make task for it.

Reply:
I created TODO in code